source/tutorials/pd-mice.rst

```diff
@@ -709,85 +721,31 @@ We can use the new classifier in exactly the same way as the standard classifier
-     --i-taxonomy ./taxonomy.qza \
-     --o-collapsed-table ./uniform_table.qza \
-     --p-level 7 # means that we group at species level
-
-   qiime feature-table filter-features \
-     --i-table ./uniform_table.qza \
-     --p-min-frequency 50 \
-     --p-min-samples 4 \
-     --o-filtered-table ./filtered_uniform_table.qza
-
-   qiime composition add-pseudocount \
-     --i-table ./filtered_uniform_table.qza \
-     --o-composition-table ./cfu_table.qza
-
-   qiime composition ancom \
-     --i-table ./cfu_table.qza \
-     --m-metadata-file ./metadata.tsv \
-     --m-metadata-column donor \
-     --o-visualization ./ancom_donor_uniform.qzv
-
-Now redo with the new taxonomy:
+   mkdir ./multi-taxonomy
+   mv ./taxonomy.qza ./multi-taxonomy
+   mv ./bespoke_taxonomy.qza ./multi-taxonomy
 
 .. command-block::
 
-   qiime taxa collapse \
-     --i-table ./table_2k.qza \
-     --i-taxonomy ./bespoke_taxonomy.qza \
-     --p-level 7 \
-     --o-collapsed-table ./bespoke_table.qza
-
-   qiime feature-table filter-features \
-     --i-table ./bespoke_table.qza \
-     --p-min-frequency 50 \
-     --p-min-samples 4 \
-     --o-filtered-table ./filtered_bespoke_table.qza
-
-   qiime composition add-pseudocount \
-     --i-table ./filtered_bespoke_table.qza \
-     --o-composition-table ./cfb_table.qza
-
-   qiime composition ancom \
-     --i-table ./cfb_table.qza \
-     --m-metadata-file ./metadata.tsv \
-     --m-metadata-column donor \
-     --o-visualization ./ancom_donor_bespoke.qzv
+   qiime feature-table tabulate-seqs \
+     --i-data dada2_rep_set.qza \
+     --i-taxonomy multi-taxonomy/ \
+     --o-visualization dada2_rep_set_multi_taxonomy.qzv
 
 .. question::
+   Open up the ``dada2_rep_set_multi_taxonomy.qzv`` visualization and the ``da_barplot_donor.qzv`` visualization.
 
-   Compare final ANCOM visualizations. They are fairly similar, which is good.
-
-   1. Is *Bacteroides ovatus* present in the ANCOM results derived from our original taxonomy?
-   2. Is *B. ovatus* present in the new ANCOM results?
-   3. Why is that?
+   1. Examine the enriched ASVs in the ``da_barplot_donor.qzv`` visualization. Are there any of these enriched ASVs that have differing taxonomic resolution in the ``dada2_rep_set_multi_taxonomy.qzv`` visualization?
+   2. If so, which taxonomy provided better resolution?
+   3. Is this what we expect, based on what we learned about taxonomic classification, accuracy, and re-training earlier in the tutorial?
 
-.. no
-.. yes
-.. The original taxonomy lumped it in with g__Bacteroides; __ and the effect was washed out.
+.. Yes: 04195686f2b70585790ec75320de0d6f, 54f7ee881a58ad84fe3f81d76968b072, d2d1d9d57e61a764383ea2c84cef04c5
+.. 04195686f2b70585790ec75320de0d6f (bespoke), 54f7ee881a58ad84fe3f81d76968b072 (original), d2d1d9d57e61a764383ea2c84cef04c5 (bespoke)
+.. Bespoke provided slightly better resolution overall, which is reasonable since that was the taxonomy that we trained using animal stool samples.
```

Comment:
```suggestion
.. Bespoke provided slightly better resolution overall, which is reasonable since that was the classifier that we trained using animal stool samples.
```

This is stupid and nit-picky but I think you can train classifiers and not taxonomies.  A classifier is a machine learning model that can be trained, while a taxonomy is a static result